velox/exec/Window.cpp

```diff
@@ -505,9 +537,25 @@ RowVectorPtr Window::getOutput() {
     result->childAt(i) = output;
   }
 
-  // Compute the output values of window functions.
-  callApplyLoop(numOutputRows, result);
-  return result;
+  if (dynamic_cast<StreamingWindowBuild*>(windowBuild_.get())) {
+    // Split the output size based on the numRowsPerOutput_.
+    auto beforeApplyLoopProcessedRows = numProcessedRows_;
```

Comment:
@aditi-pandit  Assuming the RowVector has 40 rows, and after calling addInput(), the partitionStartRows is [0, 16, 32]. The first getOutput() call will only process 32 rows for the result, while the remaining 8 rows will be handled in the next getOutput() call. Therefore, we cannot guarantee that the result will be equal to numRowsPerOutput_ if not changing the getOutput().